src/store/RelayGarbageCollection.js

```diff
@@ -1,104 +0,0 @@
-/**
```

Comment:
We're using this module, and outright deleting it will make importing this PR more difficult. Let's keep the module in place, but change it to call the corresponding functions on `RelayStore` (which should be an instance of `RelayContext`).

```
const RelayGarbageCollection = {
  initialize(stepLength: number) {
    RelayStore.initialize(stepLength);
  }
  ...
}
```